packages/zoltan2/test/connectivity/meshData/noedges-basal-friction.ascii

```diff
@@ -0,0 +1,5 @@
+4
```

Comment:
This file says all four vertices are grounded.  But the answers file says they should be removed.  Grounded points should not be removed, right? If the code removes them, that would be an error.
Perhaps this test should have some grounded vertices and some not-grounded vertices; the code should remove only the not-grounded vertices.

Reply:
Vertices that are initially grounded may not necessarily be kept. In this case, they are removed because there are no edges, so it is impossible for them to be sufficiently grounded.